cloud/aws/README.md

```diff
@@ -1,10 +1,126 @@
-# VAST cloud deployment
+# VAST AWS deployment
 
-Deploy VAST easily to AWS with Terraform.
+*Disclaimer: the native cloud architecture is still highly experimental and
+subject to change without notice. Do not use in production yet.*
 
-## Proposed architecture
+Running VAST on AWS relies on two serverless building blocks:
 
-### Infrastructure overview
+1. [Fargate](https://aws.amazon.com/fargate/) for VAST server nodes
+2. [Lambda](https://aws.amazon.com/lambda/) as compute service to perform
+   tasks, such as executing queries and ingesting data ad hoc
+
+For storage on the VAST server in Fargate, VAST uses
+[EFS](https://aws.amazon.com/efs/) as a drop-in network filesystem. It provides
+persistence of the database files across task executions with a reasonable
+throughput-latency trade-off.
+
+The sketch below illustrates the high-level architecture:
+
+![AWS Architecture](https://user-images.githubusercontent.com/53797/156728636-9909c4aa-34a0-47f4-b6f0-50d7bfe96e28.png)
+
+We are still evaluating local storage optimization using
+[EBS](https://aws.amazon.com/ebs/) and high-throughput scale-out using
```

Comment:
Actually for Fargate you don't have EBS as such, its just an attached "ephemeral storage". I don't know if it is worth going into these details.

Reply:
Glad you mention this; I'll remove the sentence on EBS.